mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -27,6 +27,38 @@ class _ChunkRelevance(_BaseBuiltInScorer):
 def chunk_relevance():
     """
     Chunk relevance measures whether each chunk is relevant to the input request.
+
```

Comment:
```suggestion

    You can invoke the scorer directly with a single input for testing, or pass it to
    `mlflow.genai.evaluate` for running full evaluation on a dataset.
    
```

nit: let's add some description for when to do "direct usage" vs "evaluate"